safe/gui/tools/minimum_needs/needs_calculator_dialog.py

```diff
@@ -22,14 +22,14 @@
 from PyQt4 import QtGui
 from PyQt4.QtCore import pyqtSignature, pyqtSlot, QSettings
 
-from safe.common.utilities import temp_dir
+from safe.common.utilities import temp_dir, unique_filename
 from safe.common.version import get_version
 from safe.datastore.folder import Folder
 from safe.definitions.fields import displaced_field, aggregation_name_field
 from safe.definitions.layer_purposes import layer_purpose_aggregation
-from safe.definitions.post_processors import minimum_needs_post_processors
+from safe.definitions import minimum_needs_post_processors
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->